### ISSUE TITLE
Add skip link to every page

### ISSUE THREAD

**Issue by lhenze** [nyudlts/ultraviolet#124]
from NYU A11y review - 
A bypass block (e.g. - "Skip to Main" link) is missing from the site.  

https://www.w3.org/TR/WCAG20/#navigation-mechanisms-skip
